definition/data/sheets/AuthorisationCaseField.json

```diff
@@ -267,5 +267,7 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "expertReportInstruction", "UserRole": "caseworker-cmc-judge", "CRUD": "CRUD"},
   {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "dateReferredForDirections", "UserRole": "caseworker-cmc-judge", "CRUD": "CRUD"},
   {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "dateReferredForDirections", "UserRole": "citizen", "CRUD": "CRUD"},
-  {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "dateReferredForDirections", "UserRole": "caseworker-cmc", "CRUD": "CRUD"}
+  {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "dateReferredForDirections", "UserRole": "caseworker-cmc", "CRUD": "CRUD"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "generalLetterContent", "UserRole": "caseworker-cmc", "CRUD": "CRUD"},
```

Comment:
Should we not make these read-only, or else the event will show in production before the actual release date?

Reply:
Yes Chris they will be read only..just changed for time being to regression test to make sure nothing breaks after the review comments are addressed.